lib/server/services/dataPasses/DataPassService.js

```diff
@@ -139,6 +142,37 @@ class DataPassService {
         };
     }
 
+    /**
+     * Set given production as skimmable
+     *
+     * @param {DataPassIdentifier} identifier identifier of data pass
+     * @return {Promise<void>} resolved once the production was marked as skimmable
+     */
+    async setAsSkimmable(identifier) {
+        const validSkimmableProductionNameRegex = /_apass\d+(?!.*(skimming|skimmed))/;
+
+        return dataSource.transaction(async () => {
+            const dataPass = await this.getOneOrFail(identifier);
+            if (dataPass.pdpBeamType !== PdpBeamType.PROTON_PROTON) {
+                throw new BadParameterError(`Cannot mark ${dataPass.name} as skimmable.`
+                    + ' Only production for PROTON_PROTON runs can be marked as skimmable');
+            }
+            if (!validSkimmableProductionNameRegex.test(dataPass.name)) {
+                throw new BadParameterError(`Cannot mark ${dataPass.name} as skimmable. Only \`apass\` can be marked as skimmable`);
+            }
+
+            const dataPassDB = await DataPassRepository.findOne({ where: { id: dataPass.id } });
```

Comment:
Why DB suffix?

Reply:
Convention like in MonAlisa Synchronizer, to indicate it's sequelize model instance, but not an entity